HelloWorldServlet/src/example/HelloWorld.java

```diff
@@ -0,0 +1,54 @@
+package example;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Servlet implementation class HelloWorld
+ */
+@WebServlet("/SimplExample")//servlet url
+public class HelloWorld extends HttpServlet {
+	private static final long serialVersionUID = 1L;
+       
+    public HelloWorld() {
```

Comment:
You don`t use this method, so i think you can remove it

Reply:
it is not a method :) It is a constructor ;)

Reply:
oh, yes, my bad